pkg/apis/operator/v1alpha1/tektonpipeline_defaults.go

```diff
@@ -150,6 +150,17 @@ func (p *Pipeline) setDefaults() {
 		p.EnableGitResolver = ptr.Bool(true)
 	}
 
+	// Statefulset Ordinals
+	// if StatefulSet Ordinals mode, buckets should be equal to replicas
+	if p.Performance.StatefulsetOrdinals == nil {
+		p.Performance.StatefulsetOrdinals = ptr.Bool(false)
```

Comment:
I think we might not need to set this to default value false when it is nil

Reply:
yes true i think we can omit this